eslint warning in stickyEvents

`prepublishOnly` has this warning:

```
 warning  in ./src/lib/stickyEvents.js

Module Warning (from ./node_modules/thread-loader/dist/cjs.js):

/Users/home/Sites/olets/tailwind-config-viewer/src/lib/stickyEvents.js
  69:7  error  Unexpected console statement  no-console
```

The full eslint context is

```
Module Warning (from ./node_modules/eslint-loader/index.js):
error: Unexpected console statement (no-console) at src/lib/stickyEvents.js:69:7:
  67 |   enableEvents () {
  68 |     if (window.self !== window.top) {
> 69 |       console.warn('StickyEvents: There are issues with using IntersectionObservers in an iframe, canceling initialization. Please see https://github.com/w3c/IntersectionObserver/issues/183')
     |       ^
  70 |
  71 |       return
  72 |     }
```

--- a/src/lib/stickyEvents.js
+++ b/src/lib/stickyEvents.js
@@ -66,6 +66,7 @@ export default class StickyEvents {
 
   enableEvents () {
     if (window.self !== window.top) {
+      // eslint-disable-next-line
       console.warn('StickyEvents: There are issues with using IntersectionObservers in an iframe, canceling initialization. Please see https://github.com/w3c/IntersectionObserver/issues/183')
 
       return

